dom/src/main/javacc/org/apache/james/mime4j/field/datetime/DateTimeParser.jj

```diff
@@ -225,7 +225,15 @@ int month() :
 String year() :
 {Token t;}
 {
-    t=<DIGITS> { return t.image; }
+    t=<DIGITS> 
+    {
+      String year = (String) t.image;
+      if (year.length() == 2) 

```

Comment:
This means that the mail is 18 years old...
Do we have an added value to handle such cases?